src/utils/BucketUtils.js

```diff
@@ -99,7 +98,7 @@ export const binCollectionToBuckets = async (collection, dacIds = []) => {
 
     // Step 1.b: Populate translated dataUses
     if (dataset.dataUse) {
-      const index = findIndex(dataset.dataUse)(rawDataUses);
+      const index = rawDataUses.findIndex(du => du === dataset.dataUse);
```

Comment:
I don't know how you feel about this generally, but my preference is to use the ES6+ built-ins instead of `lodash` instead precisely for these reasons.

Reply:
> I don't know how you feel about this generally, but my preference is to use the ES6+ built-ins instead of `lodash` instead precisely for these reasons.

I'm starting to lean in that same direction - it's making more sense.